README.md

```diff
@@ -21,45 +21,31 @@ The Databricks SDK for Go includes functionality to accelerate development with
 
 During the Beta period, you must clone and then reference this repository locally, as follows. After the Beta period, you will be able to download and reference the Databricks SDK for Go package by using familiar commands such as `go get` and `go install`.
 
-1. On your local development machine with Go already [installed](https://go.dev/doc/install) and a Go code [project](https://go.dev/doc/code) active, clone this repository into your existing Go code project's current working directory, by running the `git clone` command:
-
-   ```bash
-   git clone git@github.com:databricks/databricks-sdk-go.git
-   ```
-   
-   This command creates a folder named `databricks-sdk-go` within your project. This folder contains a local copy of the Databricks SDK for Go source code.
-   
-2. Create a `go.mod` file to track your Go code's dependencies by running the `go mod init` command, for example:
+1. On your local development machine with Go already [installed](https://go.dev/doc/install) and a Go code [project](https://go.dev/doc/code) active, create a `go.mod` file to track your Go code's dependencies by running the `go mod init` command, for example:
 
    ```bash
    go mod init sample
    ```
    
-3. Take a dependency on the Databricks SDK for Go package by running the `go mod edit -require` command:
+2. Take a dependency on the Databricks SDK for Go package by running the `go mod edit -require` command:
 
    ```bash
-   go mod edit -require github.com/databricks/databricks-sdk-go@v0.0.0
+   go mod edit -require github.com/databricks/databricks-sdk-go@v0.1.0
    ```
    
-4. Redirect the dependency to your locally downloaded copy of the Databricks SDK for Go by running the `go mod edit -replace` command:
-
-   ```bash
-   go mod edit -replace github.com/databricks/databricks-sdk-go@v0.0.0=./databricks-sdk-go/ 
-   ```
-
    Your `go.mod` file should now look like this:
    
    ```go
    module sample
 
    go 1.18
 
-   require github.com/databricks/databricks-sdk-go v0.0.0
+   require github.com/databricks/databricks-sdk-go v0.1.0
 
-   replace github.com/databricks/databricks-sdk-go v0.0.0 => ./databricks-sdk-go/
+   // indirect dependencies
```

Comment:
This comment is not present in my `go.mod` file. Should this be removed to match, just to be safe?